GCC 13 has stricter allocator checks:
https://gcc.gnu.org/gcc-13/porting_to.html

Also, libstdc++ no longer leaks stdint.h definitions.
--- a/lib/mathlib.cpp
+++ b/lib/mathlib.cpp
@@ -25,2 +25,3 @@
 #include <cmath>
+#include <cstdint>
 #include <cstdlib>
--- a/lib/smallvector.h
+++ b/lib/smallvector.h
@@ -43,2 +43,5 @@
     {}
+
+    template<class U> TaggedAllocator(const TaggedAllocator<U, N>);
+    template<class U> struct rebind { using other = TaggedAllocator<U, N>; };
 };
